src/projectDB/projectdatabase.cpp

```diff
@@ -527,6 +539,225 @@ void ProjectDatabase::saveScans(const vector<mzSample*>& sampleSet)
+                      , :max_mz                           \
+                      , :min_rt                           \
+                      , :max_rt                           \
+                      , :min_intensity                    \
+                      , :max_intensity                    \
+                      , :database_search                  \
+                      , :compound_extraction_window       \
+                      , :match_rt                         \
+                      , :compound_rt_window               \
+                      , :limit_groups_per_compound        \
+                      , :match_fragmentation              \
+                      , :min_frag_match_score             \
+                      , :fragment_tolerance               \
+                      , :min_frag_match                   \
+                      , :report_isotopes                  \
+                      , :peak_quantitation                \
+                      , :min_group_intensity              \
+                      , :intensity_quantile               \
+                      , :min_group_quality                \
+                      , :quality_quantile                 \
+                      , :min_signal_blank_ratio           \
+                      , :signal_blank_ratio_quantile      \
+                      , :min_signal_baseline_ratio        \
+                      , :signal_baseline_ratio_quantile   \
+                      , :min_peak_width                   \
+                      , :min_good_peak_count              \
+                      , :peak_classifier_file             \
+                      , :alignment_good_peak_count        \
+                      , :alignment_limit_group_count      \
+                      , :alignment_peak_grouping_window   \
+                      , :alignemnt_wrt_expected_rt        \
+                      , :alignment_min_peak_intensity     \
+                      , :alignment_min_signal_noise_ratio \
+                      , :alignment_min_peak_width         \
+                      , :alignment_peak_detection         \
+                      , :poly_fit_num_iterations          \
+                      , :poly_fit_polynomial_degree       \
+                      , :obi_warp_reference_sample        \
+                      , :obi_warp_show_advance_params     \
+                      , :obi_warp_score                   \
+                      , :obi_warp_response                \
+                      , :obi_warp_bin_size                \
+                      , :obi_warp_gap_init                \
+                      , :obi_warp_gap_extend              \
+                      , :obi_warp_factor_diag             \
+                      , :obi_warp_factor_gap              \
+                      , :obi_warp_no_standard_normal      \
+                      , :obi_warp_local                   \
+                      , :main_window_compound_db_selected \
```

Comment:
Column name in schema.h is "main_window_selected_db_name". Please make it consistent to prevent confusion in the future.

Reply:
Will correct. This doesn't just cause confusion, I'd be surprised if the db name was being saved at all.